### PR TITLE
i18n: "still reachable"

### PR DESCRIPTION
**Pull request by Alkarex**
fix wording
https://github.com/FreshRSS/FreshRSS/pull/7030#discussion_r1866650296
and address risk of confusion
https://github.com/FreshRSS/FreshRSS/issues/7025#issuecomment-2509400455

### CODE REVIEW

File: app/i18n/en/sub.php [hunk 1]
```diff
@@ -67,7 +67,7 @@
 		),
 		'description' => 'Description',
 		'empty' => 'This feed is empty. Please verify that it is still maintained.',
-		'error' => 'This feed has encountered a problem. Please verify that it is always reachable then update it.',
+		'error' => 'This feed has encountered a problem. If this problem persists, If this problem persists, please verify that it is still reachable.',
```
Comment (Frenzie) [inline app/i18n/en/sub.php]:
```suggestion
		'error' => 'This feed has encountered a problem. If this problem persists, please verify that it is still reachable.',
```
I'm not completely happy about the non-typo repetition of problem though. Maybe something like this?
```suggestion
		'error' => 'This feed has encountered a problem. If this situation persists, please verify that it is still reachable.',
```

Comment (Alkarex) [reply app/i18n/en/sub.php]: https://github.com/FreshRSS/FreshRSS/pull/7049/commits/6529137f0c8ea394e87a981fe72b678df4c0f098

**Review by Frenzie — APPROVED** [review]
:+1:

File: app/i18n/nl/sub.php [hunk 2]
```diff
@@ -67,7 +67,7 @@
 		),
 		'description' => 'Omschrijving',
 		'empty' => 'Deze feed is leeg. Controleer of deze nog actueel is.',
-		'error' => 'Deze feed heeft problemen. Verifieer a.u.b het doeladres en actualiseer het.',
+		'error' => 'Deze feed heeft problemen. Verifieer a.u.b het doeladres.',	// DIRTY
```
Comment (Frenzie) [inline app/i18n/nl/sub.php]:
```suggestion
		'error' => 'Deze feed heeft een probleem ondervonden. Als deze situatie aanhoudt, controleer dan of de feed nog steeds bereikbaar is.',
```

**Pull request merged**